### DISCUSSION TITLE
BuildUtils - Add support for pnpm package manager

### DISCUSSION THREAD

**Discussion by RomainBillot** [vercel/vercel#7362] [Ideas] [answered]
Hi,

I am trying to use a package [tf-next](https://github.com/milliHQ/terraform-aws-next-js/tree/main/packages/tf-next) that depends on `@vercel/build-utils`.
Mu current package manager is `pnpm` and for an unknown reason, the `build-utils` package try to invoke some commands using `yarn` package manager.

Is it possible to add support for other package managers like `pnpm`?

Thanks in advance.

**Comment by styfle** [comment, marked as answer]
Yes, pnpm support is available now!

https://vercel.com/changelog/projects-using-pnpm-can-now-be-deployed-with-zero-configuration
